server/scripts/dbPopulate.js

```diff
@@ -0,0 +1,67 @@
+const mongoose = require('mongoose')
+const tripSchema = require('../models/TripModel')
+const stageSchema = require('../models/StageModel')
+const userSchema = require('../models/UserModel')
+const { ObjectId } = require('mongodb')
+
+const conString =
+  'mongodb+srv://bugstorm1000:GqHIshuc7OuDPDd1@cluster0.4bflh41.mongodb.net/BugStormDEV_Rithin?retryWrites=true&w=majority'
+
+// Connect to Mongo DB
+mongoose.connect(conString, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+})
+
+// Create models
+const Trip = mongoose.model('Trip', tripSchema.schema)
+const Stage = mongoose.model('Stage', stageSchema.schema)
+const User = mongoose.model('User', userSchema.schema)
+
+async function manageData() {
+  // Delete all records from trips, stages and users collections
+  await Trip.deleteMany({})
```

Comment:
are we able to call meng's resetDB script first instead? then we may not to drop here again